.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
+        required: false
+        type: string
+        description: |
+          relative path (to repo root) to YAML/Json file containing definition of
+          component-references added to base-component-descriptor, defaults to
+          `.ocm/component-references.yaml`.
+          action will only raise on absence of file if an explicit value is given.
```

Comment:
nit: I think the term `raise` is common only in Python - in other contexts, I think it is more adequate to use terms like `fail` or `error` (also, there is technically no means to raise an exception from a GitHub-Action..)